src/test/java/training/article/create.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Create an article
+
+  Background:
+    * url url
+    * def login = call read('../sign_in/sign_in.feature@loginSuccessfully')
+    * def username = username
+
+    * def token = "Bearer " + login.response.user.token
+
+  @createSuccessfully
```

Comment:
remember what we talked about the tags. You must to keep a logic on the names and add other names if you have multiples scenarios in a feature and you want to run only one.